### PR TITLE
Ft domain skeleton

### PR DESCRIPTION
**Pull request by SukafuDesu**

### CODE REVIEW

File: FakebookNotifications/FakebookNotifications.Domain/Models/User.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        /// <summary>
+        /// Checks whether a user is a valid object or not.
+        /// </summary>
+        /// <returns>
+        /// A boolean.
+        /// </returns>
+        public bool IsValid()
+        {
+            return !(string.IsNullOrEmpty(Id) || string.IsNullOrEmpty(Email));
```
Comment (ghost) [inline FakebookNotifications/FakebookNotifications.Domain/Models/User.cs]: We don't want implementation in this PR. Instead have it throw not implemented for all methods.

**Review by ghost — CHANGES_REQUESTED** [review]
There should be tests for domain methods that fail.

**Comment by ghost** [comment]
The docker build also failed in the pipeline.

**Review by ghost — APPROVED** [review]
This looks good. I tested it and the tests do fail.

**Review by mattmarnien — APPROVED** [review]

File: FakebookNotifications/FakebookNotifications.Domain/Models/User.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                {
+                    _id = value;
+                }
+                else
+                {
+                    // assigns the value to an empty string
+                    _id = "";
+                }
+            }
+
```
Comment (townerr) [inline FakebookNotifications/FakebookNotifications.Domain/Models/User.cs]: fix brace formatting

File: FakebookNotifications/FakebookNotifications.Testing/FakebookNotifications.Testing.csproj [hunk 3]
```diff
@@ -23,6 +23,7 @@
 	<ItemGroup>
```
Comment (townerr) [inline FakebookNotifications/FakebookNotifications.Testing/FakebookNotifications.Testing.csproj]: item group tabbed in too far

File: FakebookNotifications/FakebookNotifications.sln [hunk 4]
```diff
@@ -6,6 +6,8 @@ MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakebookNotifications.WebApi", "FakebookNotifications.WebApi\FakebookNotifications.WebApi.csproj", "{7182C800-A74A-4D44-9682-F9171BAD981C}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakebookNotifications.Testing", "FakebookNotifications.Testing\FakebookNotifications.Testing.csproj", "{3D40AD3A-1E08-4135-867A-AAD78677079D}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakebookNotifications.Domain", "FakebookNotifications.Domain\FakebookNotifications.Domain.csproj", "{10727CC8-E023-4F66-8510-151B65218FD0}"
```
Comment (townerr) [inline FakebookNotifications/FakebookNotifications.sln:10]: end project for this

**Review by townerr — CHANGES_REQUESTED** [review]

**Review by townerr — APPROVED** [review]

**Review by RevatureGentry — APPROVED** [review]

**Pull request merged**